src/main/java/org/jenkinsci/plugins/ewm/actions/ExwsAllocateAction.java

```diff
@@ -0,0 +1,46 @@
+package org.jenkinsci.plugins.ewm.actions;
+
+import hudson.model.Action;
+import org.jenkinsci.plugins.ewm.Messages;
+import org.jenkinsci.plugins.ewm.steps.model.ExternalWorkspace;
+import org.jenkinsci.plugins.workflow.graph.FlowNode;
+
+/**
+ * {@link Action} implementation for the {@link org.jenkinsci.plugins.ewm.steps.ExwsAllocateStep}.
+ * Stores the allocated {@link ExternalWorkspace}.
```

Comment:
Maybe. Not so important for the prototyping stage, when the things may change. Detailed documentation may be a burden at this point